src/iuse_actor.cpp

```diff
@@ -1566,6 +1566,44 @@ long fireweapon_on_actor::use( player *p, item *it, bool t, const tripoint& ) co
     return it->type->charges_to_use();
 }
 
+void manualnoise_actor::load( JsonObject &obj )
+{
+    use_message         = obj.get_string( "use_message" );
+    noise_message       = obj.get_string( "noise_message", "" );
+    noise               = obj.get_int( "noise", 0 );
+    moves               = obj.get_int( "moves", 0 );
+}
+
+iuse_actor *manualnoise_actor::clone() const
+{
+    return new manualnoise_actor( *this );
+}
+
+long manualnoise_actor::use( player *p, item *it, bool t, const tripoint& ) const
+{
+    if( t ) {
+        return 0;
+    }
+
+    p->moves -= moves;
+    {
+        if( noise > 0 ) {
+            sounds::sound( p->pos(), noise, _(noise_message.c_str()) );
```

Comment:
You allow an empty string as `noise_message`, translating an empty string does not result in an empty string. It "translates" to some project/translation info, you can see that in #10882, which was caused by translating an empty string.

I suggest this:
```C++
sounds::sound( p->pos(), noise, noise_message.empty() ? "" : _(noise_message.c_str()) );
```